x-pack/plugins/ml/server/models/data_recognizer/data_recognizer.ts

```diff
@@ -201,40 +208,57 @@ export class DataRecognizer {
       })
     );
 
-    return configs;
+    const soConfigs = (await this._loadSavedObjectModules()).map((module) => ({
```

Comment:
I'd avoid abbriviation here
```suggestion
    const savedObjectsConfigs = (await this._loadSavedObjectModules()).map((module) => ({
```

Reply:
https://github.com/elastic/kibana/pull/92855/commits/961396d595104dfe9d633151737a81fde9764366